deployment/network-operator/templates/operator.yaml

```diff
@@ -27,8 +30,15 @@ spec:
       {{- include "network-operator.selectorLabels" . | nindent 6 }}
   template:
     metadata:
+      {{- if .Values.operator.admissionController.enabled }}
+      annotations:
+        kubectl.kubernetes.io/default-container: {{ .Chart.Name }}
```

Comment:
im fine with always adding this one, one less IF statement. WDYT ?

Reply:
done